extensions/kafka-connect/src/test/java/com/hazelcast/jet/kafka/connect/KafkaConnectJdbcIntegrationTest.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2023 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.kafka.connect;
+
+import com.hazelcast.config.Config;
+import com.hazelcast.jet.Job;
+import com.hazelcast.jet.config.JobConfig;
+import com.hazelcast.jet.core.JetTestSupport;
+import com.hazelcast.jet.pipeline.Pipeline;
+import com.hazelcast.jet.pipeline.Sinks;
+import com.hazelcast.jet.pipeline.StreamStage;
+import com.hazelcast.jet.pipeline.test.AssertionCompletedException;
+import com.hazelcast.jet.pipeline.test.AssertionSinks;
+import com.hazelcast.test.annotation.ParallelJVMTest;
+import com.hazelcast.test.annotation.SlowTest;
+import org.apache.kafka.connect.data.Values;
+import org.junit.ClassRule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.testcontainers.containers.MySQLContainer;
+
+import java.net.URL;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.Statement;
+import java.util.Properties;
+import java.util.concurrent.CompletionException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+@Category({SlowTest.class, ParallelJVMTest.class})
+public class KafkaConnectJdbcIntegrationTest extends JetTestSupport {
+
+    private static final int ITEM_COUNT = 10_000;
+    private static final String USERNAME = "mysql";
+    private static final String PASSWORD = "mysql";
+    private static final String CONNECTOR_URL = "https://hazelcast.jfrog.io/artifactory/download-local"
+            + "/tests/confluentinc-kafka-connect-jdbc-10.6.3.zip";
```

Comment:
Can we distribute the jar in this way?

Reply:
I'm not a license expert. Here's the download page: https://www.confluent.io/hub/confluentinc/kafka-connect-jdbc. The connector is released under [Confluent Community License](https://www.confluent.io/confluent-community-license):

>Agreement to: (a) use the Software; (b) prepare modifications and derivative works of the Software; **(c) distribute the Software (including without limitation in source code or object code form)**; and (d) reproduce copies of the Software (the “License”). Licensee is not granted the right to, and Licensee shall not, exercise the License for an Excluded Purpose. 

Is this a blocker for bundling connector with Viridian?

> For purposes of this Agreement, “Excluded Purpose” means making available any software-as-a-service, platform-as-a-service, infrastructure-as-a-service or other similar online service that competes with Confluent products or services that provide the Software.

Reply:
I think it'll be a blocker for Viridian, but please doublecheck with the legal dep.